README.md

```diff
@@ -1,40 +1,67 @@
+mm-webapp: `feature/cloud-apps`  
+mm-plugin-apps: `master`
 
-**Current Branches Needed**
+## Quick Start
 
-The current master branches have not been merged to work with this app and the following are needed until they are mergd with master
+### Zendesk and Mattermost Users (System Admin privileges required)
+
+1. Clone this repo
+1. cp `.env.sample` `.env`
+    1. `ZENDESK_URL` - set URL to your zendesk account host
+        1. Ex. `https://<subdomain>.zendesk.com`
+    1. `ZENDESK_CLIENT_SECRET` - (will be set later in the setup)
+1. Create Zendesk Oauth Client for Mattermost (in Zendesk)
+    1. `Zendesk` > `Admin` > `API` > `OAuth Clients`
+    1. `Add OAuth Client`
+        1. `Client Name`: `Mattermost Zendesk App`
+        1. `Description`: `Connect your Zendesk to accout to Mattermost`
+        1. `https://<your-zendesk-app-host>/mattermost/oauth/complete`
+            1. Ex. `https://localhost:4000` - Development
+            1. Ex. `https://mytest.ngrok.io` - Exposed for development
+        1. `Secret` - Save the generated secret in `.env` as the `ZENDESK_CLIENT_SECRET`
+        1. `Save`
+1. Start the node server
+    1. `npm start` - start the node server
+    1. `npm run build` - builds the dist dir.  (TODO: can this be automated?  Fails if this is not run before `npm start`)
+    1. `npm run build:watch` (in separate shell) - watch for changing files and report typescript errors
+1. Install the app (In Mattermost)
+    1. `/apps install --url http://<your-zendesk-app-host>/mattermost/manifest.json --app-secret thisisthesecret`  
+        1. TODO this command could be used to set zendesk-url and zendesk-client-secret
+
+### Zendesk and Mattermost Users (All users)
+
+#### Connect your Zendesk account to Mattermost
+
+- `/zendesk connect`
+
+This slash command will allow a user to connect their Mattermost and Zendesk
+accounts via OAuth2 authorization
+
+## Slash Commands
+
+`/zendesk connect` - connect your Zendesk account to Mattermost  
+`/zendesk disconnect` - disconnect your Zendesk account from Mattermost
 
-mm-webapp: `feature/cloud-apps`  
-mm-plugin-apps: `master`
+## Create a ticket
 
-**Install the app**
+Creating a ticket from a Mattermost post is done through the `...` post menu button
 
-`/apps install --url http://localhost:4000/manifest.json --app-secret thisisthesecret`
+## Installation
 
-  After installing the app, a provisioned bot account will be created for user @zendesk and posted in a DM.
+`/apps install --url http://<your-zendesk-app>/mattermost/manifest.json --app-secret thisisthesecret`  
+`/apps install --url http://localhost:4000/mattermost/manifest.json --app-secret thisisthesecret`  
 
-  The following values are stored locally in `app_store.json`
+After installing the app, a provisioned bot account will be created for user `@zendesk` and posted in a DM. The following values are stored locally in `config.json`
```

Comment:
in config.json or the config store? (maybe we should avoid using the config.json name and use it only for the server config file)

Reply:
I'm 0/5 on the naming decision.  `config.json` shouldn't create confusion, as this app doesn't have the concept of the Mattermost server.  I have ticket to convert to store in the plugin KV store.  This is just temporary for now.

Reply:
ticket created https://mattermost.atlassian.net/browse/MM-31907